### PR TITLE
add zh_TW translate

### PR DESCRIPTION
**Pull request by ibmibmibm**
## Guidelines for Contributing

Libresonic development is a community project, and contributions are welcomed. Here are a few guidelines you should follow before submitting:
1.  **License Acceptance** All contributions must be licensed as [GNU GPLv3](https://github.com/Libresonic/libresonic/blob/develop/LICENSE.txt) to be accepted. Use [`git commit --signoff`](https://jk.gs/git-commit.html) to acknowledge this.
2.  **No Breakage** New features or changes to existing ones must not degrade the user experience. This means do not introduce bugs, remove functionality, or make large changes to existing themes/UI without prior discussion in an Issue.
3.  **Coding standards** Language best-practices should be followed, comment generously, and avoid "clever" algorithms. Refactoring existing messes is great, but watch out for breakage.
4.  **Be bold!** Without contributions, this project will vanish. If you just want to help out, try [submiting a patch](https://github.com/Libresonic/libresonic/issues?q=is%3Aissue+is%3Aopen+label%3Apatches-welcome) for an open Issue.
5.  **Stay relevant** Issues or commentary that is off-topic or tangential to Libresonic development is subject to moderation. Questions should be focused on improving documentation to solve a problem. Visit [Reddit](https://www.reddit.com/r/libresonic) or [IRC](http://webchat.freenode.net?channels=%23libresonic) for community discussion.

Signed-off-by: Shen-Ta Hsieh ibmibmibm.tw@gmail.com

### CODE REVIEW

File: libresonic-main/src/main/resources/org/libresonic/player/i18n/ResourceBundle_zh_TW.properties [hunk 1]
```diff
@@ -98,46 +162,68 @@ status.current = \u76EE\u524D\u64AD\u653E
...
 gettingStarted.step2.text = \u8A2D\u5B9A\u60A8\u97F3\u6A02\u6240\u653E\u7F6E\u7684\u8CC7\u6599\u593E\u3002
-gettingStarted.step3.title = \u914D\u7F6E\u7DB2\u8DEF\u8A2D\u5B9A.
+gettingStarted.step3.title = \u914D\u7F6E\u7DB2\u8DEF\u8A2D\u5B9A\u3002
 gettingStarted.step3.text = \u5982\u679C\u60A8\u8981\u900F\u904E\u7DB2\u969B\u7DB2\u8DEF\u6216\u662F\u8207\u60A8\u7684\u670B\u53CB\u3001\u5BB6\u4EBA\u5206\u4EAB\u3002\
-                            \u4E26\u53D6\u5F97\u60A8\u7684\u5C08\u5C6C\u7DB2\u5740\u50CF\u662F<em>yourname</em>.libresonic.org.
+                            \u4E26\u53D6\u5F97\u60A8\u7684\u5C08\u5C6C\u7DB2\u5740\u50CF\u662F<em>yourname</em>.libresonic.org\u3002
 gettingStarted.hide = \u4E0B\u6B21\u4E0D\u986F\u793A\u672C\u9801\uFF01
-gettingStarted.hidealert = \u5982\u679C\u4E0B\u6B21\u9084\u9700\u8981\u986F\u793A\u63D0\u793A,\u8ACB\u5F9E \u8A2D\u5B9A->\u4E00\u822C \u4E2D\u52FE\u9078.
+gettingStarted.hidealert = \u5982\u679C\u4E0B\u6B21\u9084\u9700\u8981\u986F\u793A\u63D0\u793A\uFF0C\u8ACB\u5F9E \u8A2D\u5B9A\u2192\u4E00\u822C \u4E2D\u52FE\u9078\u3002
+gettingStarted.hidealert = To show this screen again, go to Settings > General.
```
Comment (muff1nman) [inline libresonic-main/src/main/resources/org/libresonic/player/i18n/ResourceBundle_zh_TW.properties]: Shouldn't this line be removed so it doesn't override the one above?

Comment (muff1nman) [inline libresonic-main/src/main/resources/org/libresonic/player/i18n/ResourceBundle_zh_TW.properties]: Fixed this.

**Comment by muff1nman** [comment]
Merged, thank you! If you happen to have time, I think there is one improvement to make:
screenshot at 2016-11-15 20-47-45

**Comment by ibmibmibm** [comment]
Ok, I'll translate more if i have time.

**Pull request closed without merging**